i3pystatus/openvpn.py

```diff
@@ -54,7 +54,13 @@ class OpenVPN(IntervalModule):
     def init(self):
         if not self.vpn_name:
             raise Exception("vpn_name is required")
-
+        
+        # use new service names if openvpn version is 2.4 (2.5 is not out yet...)
+        if run_through_shell('openvpn --version | grep -i "openvpn 2.4"', True).out.strip():
```

Comment:
You could add it as a module paramater with a default of false, that way you won't break existing configs and people can still use the feature.